changelog/v1.24.2-patch2/upgrade-alpine.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
+- type: DEPENDENCY_BUMP
+  description: Update alpine glibc for scanners that dont account for our apk upgrades. Eventually we will move to distroless like upstream
+  dependencyOwner: alpine-glibc
+  dependencyRepo: frovald
+  dependencyTag: 7590d4e652d5a43834384b640652a6008aebf9d1f9a7400ade09b29daffd5aa2
```

Comment:
Its the one we have in upstream but we intend to remove it entirely to reflect upstream's change but it will entail a few changes in the gloo repo too